pkg/scheduler/backend/cache/cache_test.go

```diff
@@ -40,6 +40,11 @@ import (
 	schedutil "k8s.io/kubernetes/pkg/scheduler/util"
 )
 
+var nodeInfoCmpOpts = []cmp.Option{
+	cmp.AllowUnexported(framework.NodeInfo{}),
+	cmpopts.IgnoreUnexported(framework.PodInfo{}),
```

Comment:
why we use AllowUnexported for one and IgnoreUnexported for another? can we use either of them?